db/column_family.cc

```diff
@@ -829,7 +829,7 @@ WriteStallCondition ColumnFamilyData::RecalculateWriteStallConditions(
           name_.c_str(), vstorage->estimated_compaction_needed_bytes(),
           write_controller->delayed_write_rate());
     } else {
-      assert(write_stall_condition == WriteStallCondition::kNormal);
+      assert(write_stall_condition == WriteStallCondition::kNormal || mutable_cf_options.disable_write_stall);
```

Comment:
@Connor1996 Minor: should be `else if (write_stall_condition == WriteStallCondition::kNormal)` ?

Reply:
What's the difference

Reply:
Each branch has its own precondition, which is "`write_stall_condition` is unset" here. For this specific spot, breaking the precondition might not cause any trouble, but it's a mistake nonetheless.

Reply:
`disable_write_stall` is a kind of "write_stall_condition is unset", using `else if` needs to duplicate the logic in `else` branch.